runtimes/nodejs/nodejs.md

```diff
@@ -185,22 +185,34 @@ it from your environment: `process.env.MY_VARIABLE`.
 You can (and should) define the `NODE_ENV=production` environment variable as many frameworks
 rely on it for production mode.
 
-### Special env NPM_TOKEN
 
-Since April 2015, npmjs.com allows you to have private repositories. The bad news is,
-you have to write a specific line in your `.npmrc` for that. The good news is, on Clever
-Cloud, you only need to provide the *token* part, and we set up everything for you!
+### Use private repositories with NPM_REGISTRY and NPM_TOKEN
 
-So to register your npm auth token, you need to add to your application the `NPM_TOKEN`
-environment variable.
+On Clever Cloud, you only need to provide the *token* part. So to register your auth token, you need 
```

Comment:
Thats not really true anymore with CC_NPM_REGISTRY. I'd better see a text saying `To configure authentication on NPM, you can use NPM_TOKEN=token to configure the authentication token`. And then, `If you need to authenticate to another registry (github or a private one), you can also use CC_NPM_REGISTRY=<url> to configure the endpoint`

Something like that.